src/Pages/Shared/ConfigMapSecret/secret.utils.tsx

```diff
@@ -402,6 +401,29 @@ export async function prepareSecretOverrideData(configMapSecretData, dispatch: (
     }
 }
 
+export const transformSecretDataJSON = (jsonObj: any[]) => {
```

Comment:
Suggestion: this could have been typed but can be ignored since this is commoned out